pyston/data_processor.py

```diff
@@ -108,26 +106,34 @@ def _process_file_data(self, data, files, key, data_item, file_content):
             )
             data[key] = filename
         else:
-            self.errors[key] = ugettext('Unsupported file type')
+            self.errors[key] = RESTValidationError(ugettext(
```

Comment:
ty hlášky bych si ukládal do nějaký konstanty, ať se to neduplikuje, taky tu chybí gettext